gradle/repositories/build.gradle

```diff
@@ -3,4 +3,8 @@ repositories {
     jcenter()
     maven { url 'https://oss.sonatype.org/content/repositories/releases/' }
     maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
+    maven { url "https://hyperledger-org.bintray.com/besu-repo" }
```

Comment:
Do we require all of these?

Reply:
These are required yes. Removing these URLs will result in Gradle not being able to find the dependencies.